php/php.phpunit/src/org/netbeans/modules/php/phpunit/ui/options/Bundle.properties

```diff
@@ -59,6 +59,7 @@ PhpUnitOptionsPanel.phpUnitInfoLabel.AccessibleContext.accessibleName=PHPUnit cl
 PhpUnitOptionsPanel.noteLabel.AccessibleContext.accessibleName=Note:
 PhpUnitOptionsPanel.skelGenLabel.text=&Skeleton Generator Script:
 PhpUnitOptionsPanel.phpUnitLearnMoreLabel.text=<html><a href="#">Learn more about PHPUnit</a></html>
-PhpUnitOptionsPanel.skelGenLearnMoreLabel.text=<html><a href="#">Learn more about Skeleton Generator</a></html>
+PhpUnitOptionsPanel.skelGenLearnMoreLabel.text=<html><a href="#">Learn more about Skeleton Generator</a> (forked repo.)</html>
 PhpUnitOptionsPanel.skelGenBrowseButton.text=&Browse...
 PhpUnitOptionsPanel.skelGenSearchButton.text=S&earch...
+PhpUnitOptionsPanel.skelGenAbandonedLabel.text=<html><b>The original generator has been abandoned.</b></html>
```

Comment:
Nitpick: perhaps "The original [project]"?

Reply:
Will fix ("The original project has been abandoned."). Thanks!

Reply:
Fixed: https://github.com/apache/netbeans/compare/5b834cfeff13ca3279108fa7f0e0cba8448f5d91..2f71bb9cbc2e065360a210d7ef32caa98a0278e9